### PR TITLE
Feature/24 highlihting code after launching

### PR DESCRIPTION
**Pull request by BaranovMykola**
This branch has been created from feature/22-temporary-file (it still need code review), because this feature  are relative to #22.

### CODE REVIEW

File: src/kuzya.cpp [hunk 1]
```diff
@@ -193,6 +193,19 @@ Kuzya::Kuzya(QWidget *parent)
     settings->readODWSettings();
     settings->openLastProject();
     settings->readMainWindowState();
+    tFile = new QTemporaryFile;
```
Comment (alexchmykhalo) [inline src/kuzya.cpp]: Please, choose better name for member + use m prefix for members.

File: src/kuzya.cpp [hunk 2]
```diff
@@ -193,6 +193,19 @@ Kuzya::Kuzya(QWidget *parent)
...
+    if(settings->getIsFileReopenEnabled())
```
Comment (alexchmykhalo) [inline src/kuzya.cpp]: it better to use either "get" or "is" prefix for getters (I mean name it: getFileReopenState() or isFileReopenEnabled()). I suggest to use second one (isFileReopenEnabled()) - it is more in Qt fashion.

File: src/kuzya.cpp [hunk 3]
```diff
@@ -1118,7 +1131,9 @@ void Kuzya::slotMarginClicked(int margin, int line, Qt::KeyboardModifiers)
 **/
 bool Kuzya::slotSaveChangesNotifier(void)
 {
-    if (textEditor->isModified())
+    QString tfname = tFile->fileName();
```
Comment (alexchmykhalo) [inline src/kuzya.cpp]: Please, find better names for this variables..

File: src/kuzya.cpp [hunk 4]
```diff
@@ -1530,3 +1554,50 @@ void Kuzya::setAllIconsVisibleInMenu(bool isVisible)
 
 
 }
+
+void Kuzya::writeTemporaryFileState()
+{
+    QSettings* settings = new QSettings(QApplication::applicationDirPath()+"/settings.ini", QSettings::IniFormat);
```
Comment (alexchmykhalo) [inline src/kuzya.cpp]: We already have settings in OptionsDialog class. Could we use that one?

**Review by alexchmykhalo — CHANGES_REQUESTED** [review]

File: src/kuzya.cpp (hunk 4 above)
Comment (BaranovMykola) [reply src/kuzya.cpp]: I would like to use the same ini file. But according to your last comment i am not allowed to use OptionDialog class. So, i tried to work with this file without OptionDialog class. Am i wrong?

> We should stick to single responsibility principle (see SOLID for reference). Class called OptionsDialog - that means that it is dialog that contains UI for Kuzya options. That is its only responsibility. It should not take care of loading temporary files, dealing with main window configuration, etc. This class should serve only one purpose - manage UI for setting Kuzya options.

**Comment by BaranovMykola** [comment]
Extra instance of QSettings* in kuzya.cpp was removed. Used OptionsDialog's getter instead

**Pull request merged**